data/authors/lucas.mdx

```diff
@@ -0,0 +1,21 @@
+---
+name: Lucas
+nym: lucas
+avatar: /static/images/volunteers/lucas.jpg
+occupation: Volunteer
+company: OpenSats
+email: lucas@opensats.org
+nostr: npub176mj8c5pa6pxlmm8syv4uhmz0n8934w0pfnfddgawhycqcue69esr3qzt2
+volunteer: true
+---
+
+Lucas is the co-founder of several small businesses, some of which he is
+managing to this day. His entrepreneurial endeavors over the last 20 years
+spanned multiple countries and continents. As is usually the case with small
+business operators, he knows a little bit of everything and likes to get things
+done.
+
+Over the last couple of years, he developed a passion for Bitcoin and has been
```

Comment:
```suggestion
Because of his first-hand experience with hyperinflation and running businesses in extremely volatile economic and political environments, he developed a passion for Bitcoin and has been
```